### PR TITLE
Miscellaneous fixes discovered in end-to-end testing

### PR DESCRIPTION
**Pull request by jimbobhickville**
* spark-submit needed to deploy on the cluster
* port 80 is already bound by httpd, switch to 8080
* make sure manage script is executable
* adjust column width because neutral was removed
* make the node app fail with a useful message if --config was omitted

### CODE REVIEW

File: examples/twitter_sentiment_v2/README.md [hunk 1]
```diff
@@ -83,8 +83,9 @@ And if you just want to stop it entirely:
 # View the app
 
 Now that the app is running, you can view it in your browser by finding the public IP of your 
-gateway node and loading it in your browser.  You can also use the lava ssh proxy to set up a 
-Socks Proxy and load the website by visiting ``http://gateway-1.local``
+gateway node and loading it in your browser on port 8080 ``http://ip-address:8080``.  You can 
+also use the lava ssh proxy to set up a Socks Proxy and load the website by visiting
```
Comment (thesquelched) [inline examples/twitter_sentiment_v2/README.md]: `s/Socks/SOCKS/`?

**Comment by thesquelched** [comment]
Otherwise, :+1:

**Comment by jimbobhickville** [comment]
Fixed

**Comment by pchanne** [comment]
+1

**Pull request merged**